### ISSUE TITLE
Ability to change layer priority

### ISSUE THREAD

**Issue by jsnkuhn** [gameboy9/randoTracker#27]
Ability to change layer priority. In particular this would be useful assuming #11.

**Comment by Datyedyeguy** [comment]
I'm assuming this is layer priority in the XML for design purposes and not something done dynamically by the Restreamer, right?

**Comment by jsnkuhn** [comment]
yes, an example would be using a neutralPic as a animated gif background. Right now the players names appear behind the neutralPic so get covered by the full screen neutralPic. 
Or if we'd ever want to start overlapping elements. Like a tracker overlapping a the player names, we might would need control of what goes above and below.
